### PR TITLE
fix: console warning when using wrapperClassName prop on plain image

### PR DESCRIPTION
**Pull request by kopancek**
### Describe what this PR does

Get's rid of the following react warning in dev tools console:
```
Warning: React does not recognize the `wrapperClassName` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `wrapperclassname` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]
LGTM :)

**Review by omBratteng — COMMENTED** [review]
Could take a look at how I did it in #2444, that way you don't need to set types

File: packages/shared/src/components/cards/PostCardFooter.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const PostCardFooter = ({
           )}
           loading="lazy"
           data-testid="postImage"
-          wrapperClassName="my-2"
+          {...(isVideoType && { wrapperClassName: 'overflow-hidden' })}
```
Comment (omBratteng) [inline packages/shared/src/components/cards/PostCardFooter.tsx:53]: Was this supposed to be changed to `'overflow-hidden'` 😬

Comment (kopancek) [reply packages/shared/src/components/cards/PostCardFooter.tsx:53]: Damn, bad copy paste, thanks!

**Pull request merged**